docs-src/concepts/what-is-a-codec-server.md

```diff
@@ -33,16 +33,14 @@ Most SDKs provide example Codec Server implementation samples, listed here:
 
 #### Usage
 
-When using tctl or the Web UI to perform some operations on a Workflow Execution, you can configure the exposed Codec Server endpoints to remotely encode data sent to the Temporal Server and decode data received from the Temporal Server.
-
 When you apply custom encoding with encryption or compression on your Workflow data, it is stored in the encrypted/compressed format on the Temporal Server. For details on what data is encoded, see [Data encryption](/production-readiness/develop#data-encryption).
 
-Before you use a Codec Server to encode your data, ensure that you consider all the security implications of running codecs remotely.
-For example, codecs that perform encryption might need to be secured to prevent decryption by untrusted callers.
+To see decoded data when using CLI or the Web UI to perform some operations on a Workflow Execution, you can configure the exposed Codec Server endpoints to remotely decode data received from the Temporal Server.
 
-[Configure your Codec Server endpoints](/prod-readiness-context/how-to-set-up-codec-server) to decode the encoded data to its original format when viewed from the Web UI or tctl.
+For details on creating your Codec Server, see [Codec Server](/production-readiness/develop#how-to-set-up-codec-server).
 
-The following samples provide implementation examples for applying authentication on your Codec Server using the Go SDK.
+After you start your Codec Server, [configure your Codec Server endpoints](/production-readiness/develop#how-to-set-up-codec-server) to decode the encoded data to its original format when viewed from the Web UI or tctl.
 
-- [Codec Server](https://github.com/temporalio/samples-go/tree/main/codec-server)
-- [GRPC proxy server](https://github.com/temporalio/samples-go/tree/main/grpc-proxy)
+You can set your Codec Server endpoints to encode data sent to the Temporal Server (see [Remote data encoding](/concepts/what-is-remote-data-encoding)).
+However, before you use a Codec Server to encode your data, ensure that you consider all the security implications of running codecs remotely.
```

Comment:
The Codec Server does not encode data. I suggest dropping this warning from here, it's present on the prod readiness page.